app/assets/stylesheets/custom.scss

```diff
@@ -343,3 +343,45 @@ ul, menu, dir {
                 margin: 15px 0;
         }
 }
+
+
+// /* =========================
+//
+//      HOST DASHBOARD INDEX PAGE
+//
+//    =========================*/
+
+.dashboard {
+  text-align: center;
+  margin-top: 120px;
+}
+
+ul.edit-listings  {
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space